dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/patch/operation/DSpaceObjectMetadataRemoveOperation.java

```diff
@@ -91,9 +99,12 @@ private void remove(Context context, DSpaceObject dso, DSpaceObjectService dsoSe
             throw new IllegalArgumentException("This index (" + index + ") is not valid number.", e);
         } catch (ArrayIndexOutOfBoundsException e) {
             throw new UnprocessableEntityException("There is no metadata of this type at that index");
-        } catch (SQLException e) {
-            throw new DSpaceBadRequestException("SQLException in DspaceObjectMetadataRemoveOperation.remove " +
-                    "trying to remove metadata from dso.", e);
+        } catch (SQLException ex) {
+            String msg;
+            msg = "SQLException in DspaceObjectMetadataRemoveOperation.remove " +
+                    "trying to remove metadata from dso.";
+            log.error(msg, ex);
```

Comment:
is this log required?

Reply:
No, I've tested where it will be logged, here: https://github.com/dataquest-dev/DSpace/blob/dtq-dev/dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java#L711